### ISSUE TITLE
LAAT/lag, Impulse does not work

### ISSUE THREAD

**Issue by m0kka** [Special-Operations-Brigade/SOB-Issue-Tracker#203]
Reported by pilots today, cannot activate Impulse on the LAAT/lag, neither through scroll-wheel nor keybind

**Comment by m0kka** [comment]
should be resolved with rat's latest changes

**Comment by m0kka** [comment]
not fixed, apparently @Rat1452

**Comment by ThatArcanist** [comment]
some potential crucial info: was the pilot the physical driver or a UAV driver

**Comment by m0kka** [comment]
UAV of course

**Comment by gumbekk** [comment]
Any update on this? I think this came up again a week or two ago, not sure if anything was done yet

**Comment by ThatArcanist** [comment]
We might need to make our own impulse script or switch permanently to Legion studios

**Comment by Rat1452** [comment]
We need to look at the current impulse script to see who it assigns ownership to, or activation to. If it goes to "pilot" or the seat in which the pilot sits then were going to have to find a way to assign the UAV controller as "pilot" however if its assigned another way than big shrug. 

For context, the issue is that the UAV is technically in the pilot seat not the player, so the player is unable to activate things that are assigned to the seat

**Comment by Rat1452** [comment]
I don't think that Legion or 3AS will have a difference in this issue as it's to how its assigned
